controlplane/kubeadm/controllers/controller.go

```diff
@@ -222,6 +232,9 @@ func (r *KubeadmControlPlaneReconciler) reconcile(ctx context.Context, cluster *
 		logger.Error(err, "unable to lookup or create cluster certificates")
 		return ctrl.Result{}, err
 	}
+	if certificates.HasGenerated() {
+		conditions.MarkTrue(kcp, controlplanev1.CertificatesGeneratedCondition)
```

Comment:
Would external consumers actually care that certificates where generated or that certificates are available (or both)?

If watching externally for progress (or lack of progress), I'm wondering if it would make sense to possibly track both, rather than just if the certificates are generated.

Reply:
I will change into Available